pom.xml

```diff
@@ -52,11 +52,21 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>io.temporal</groupId>
+                <artifactId>temporal-opentracing</artifactId>
+                <version>${temporal.version}</version>
+            </dependency>
             <dependency>
                 <groupId>io.temporal</groupId>
                 <artifactId>temporal-testing</artifactId>
                 <version>${temporal.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.opentelemetry</groupId>
+                <artifactId>opentelemetry-opentracing-shim</artifactId>
+                <version>1.32.0</version>
```

Comment:
I am asking Bruno a question on Zulip on how I can access the opentelemetry version so it stays in sync with Quarkus's version.